submissions/adeyemiJuwon/database/init.sql

```diff
@@ -0,0 +1,222 @@
+
+--
+-- Dumping data for table `order_items`
+--
+
+LOCK TABLES `order_items` WRITE;
+/*!40000 ALTER TABLE `order_items` DISABLE KEYS */;
+INSERT INTO `order_items` VALUES (1,4,4,3.74,14.96),(2,1,2,9.10,18.20),(2,2,2,2.94,5.88),(2,3,4,1.66,6.64);
+/*!40000 ALTER TABLE `order_items` ENABLE KEYS */;
+UNLOCK TABLES;
+
+--
+-- Table structure for table `orders`
+--
+
+DROP TABLE IF EXISTS `orders`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!50503 SET character_set_client = utf8mb4 */;
+CREATE TABLE `orders` (
+  `order_id` int(11) NOT NULL AUTO_INCREMENT,
+  `customer_id` int(11) NOT NULL,
+  `order_date` date NOT NULL,
+  PRIMARY KEY (`order_id`),
+  KEY `fk_orders_customers_idx` (`customer_id`),
+  CONSTRAINT `fk_orders_customers` FOREIGN KEY (`customer_id`) REFERENCES `customers` (`customer_id`) ON UPDATE CASCADE
+) ENGINE=InnoDB AUTO_INCREMENT=1 DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_general_ci;
+/*!40101 SET character_set_client = @saved_cs_client */;
+
+--
+-- Dumping data for table `orders`
+--
+
+LOCK TABLES `orders` WRITE;
+/*!40000 ALTER TABLE `orders` DISABLE KEYS */;
+INSERT INTO `orders` VALUES (1,2,'2020-01-30'),(2,1,'2020-08-02'),(3,3,'2019-12-01'),(4,2,'2020-01-22'),(5,1,'2020-08-25'),(6,4,'2019-11-18'),(7,2,'2020-09-22'),(8,3,'2020-08-08'),(9,1,'2020-07-05'),(10,3,'2020-04-22');
+/*!40000 ALTER TABLE `orders` ENABLE KEYS */;
+UNLOCK TABLES;
+
+--
+-- Table structure for table `products`
+--
+
+DROP TABLE IF EXISTS `products`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!50503 SET character_set_client = utf8mb4 */;
+CREATE TABLE `products` (
+  `product_id` int(11) NOT NULL AUTO_INCREMENT,
+  `name` varchar(50) COLLATE utf8mb4_general_ci NOT NULL,
+  `description` varchar(250) COLLATE utf8mb4_general_ci NOT NULL,
+  `image` blob,
+  `unit_price` decimal(8,2) NOT NULL,
+  `category` int(11) NOT NULL DEFAULT '1',
+  `admin_user` int(11) NOT NULL DEFAULT '1',
+  `quantity_in_stock` int(11) NOT NULL,
+  `stock_level` varchar(50) COLLATE utf8mb4_general_ci NOT NULL,
+  PRIMARY KEY (`product_id`),
+  KEY `fk_products_admin_users_idx` (`admin_user`),
+  KEY `fk_products_categorys_idx` (`category`),
+  CONSTRAINT `fk_products_admin_users` FOREIGN KEY (`admin_user`) REFERENCES `admin_users` (`admin_user_id`) ON UPDATE CASCADE,
+  CONSTRAINT `fk_products_categorys` FOREIGN KEY (`category`) REFERENCES `categorys` (`category_id`) ON UPDATE CASCADE
+) ENGINE=InnoDB AUTO_INCREMENT=1 DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_general_ci;
+/*!40101 SET character_set_client = @saved_cs_client */;
+
+--
+-- Dumping data for table `products`
+--
+
+LOCK TABLES `products` WRITE;
+/*!40000 ALTER TABLE `products` DISABLE KEYS */;
+INSERT INTO `products` VALUES 
+(1,'Xiaomi Mi Notebook Pro 2019 15.6 inch Windows 10','Chinese Language Home Edition Intel Core i5 - 8250U Quad Core 1.6GHz CPU 8GB RAM 256GB SSD 1.0MP Front Camera Fingerprint Sensor',NULL,1485.83,1,2,50,'low stock'),
```

Comment:
Use URL for the images. E.g `https://cdn.mos.cms.futurecdn.net/ZihL9gqtbaXAXaDLzByrkC.jpg` point to an image